docs/backups-ondemand.md

```diff
@@ -1,38 +1,69 @@
 # Making on-demand backup
 
-To make an on-demand backup, the user should first make changes in the
-`deploy/cr.yaml` configuration file: set the `backup.enabled` key to
-`true` and [configure backup storage](backups-storage.md) in the
-`backup.storages` subsection.
+## Before you begin 
 
-When the `deploy/cr.yaml` file
-contains correctly configured keys and is applied with `kubectl` command, use
-*a special backup configuration YAML file* with the following contents:
+1. Export the namespace as an environment variable. Replace the `<namespace>` placeholder with your value:
 
-* **backup name** in the `metadata.name` key,
+    ```bash
+    export NAMESPACE = <namespace>
+    ```
 
-* **Percona Distribution for MySQL Cluster name** in the `clusterName` key,
+2. Check the configuration in the `deploy/cr.yaml` file:
 
-* **storage name** from `deploy/cr.yaml` in the `spec.storageName` key.
+    * Check that the `backup.enabled` key is set to `true`. Use the following command:
 
-* <a name="finalizers"></a>**S3 backup finalizer** set by the `metadata.finalizers.percona.com/delete-backup` key (it triggers the actual deletion of backup files from the S3 bucket when there is a manual or scheduled removal of the corresponding backup object).
+       ```bash
+       kubectl get ps <cluster-name> -n $NAMESPACE -o jsonpath='{.spec.backup.enabled}'
+       ```
 
-The example of such file is [deploy/backup/backup.yaml :octicons-link-external-16:](https://github.com/percona/percona-server-mysql-operator/blob/v{{release}}/deploy/backup/backup.yaml).
+    * [Configure backup storage](backups-storage.md) in the
+    `backup.storages` subsection.
 
-When the backup destination is configured and applied with kubectl apply -f deploy/cr.yaml command, make backup as follows:
+## Backup steps
 
-```bash
-kubectl apply -f deploy/backup/backup.yaml
-```
+To make an on-demand backup, use
+*a special backup configuration YAML file*. The example of such file is [deploy/backup/backup.yaml :octicons-link-external-16:](https://github.com/percona/percona-server-mysql-operator/blob/v{{release}}/deploy/backup/backup.yaml).
+
+You can check available options in the [Backup resource reference](backup-cr.md)
+
+Specify the following keys:
+
+* Set the `metadata.name` key to assign a name to the backup.
+* Set the `spec.clusterName` key to the name of your cluster.
+* Set the `spec.storageName` key to a storage configuration defined in your `deploy/cr.yaml` file.
+* Optionally, add the `percona.com/delete-backup` entry under `metadata.finalizers` to enable deletion of backup files from S3-compatible storage when the backup object is removed (manually or by schedule).
```

Comment:
We support not only S3 storage but also Azure, maybe it's better to use 'cloud storage' instead?